src/test/java/com/onto/javacourse/intro/findingArraysMedian/findingArraysMedianTest.java

```diff
@@ -0,0 +1,72 @@
+package com.onto.javacourse.intro.findingArraysMedian;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class FindingArraysMedianTest {
+    FindingArraysMedian  findingArraysMedian = new FindingArraysMedian ();
+
+    @Test
+     void testFindMedianPosition_EmptyArray() {
+        int[] array = {};
+        int result = findingArraysMedian.median(array);
+        assertEquals(-1, result);
+    }
+
+    @Test
+     void testFindMedianPosition_SingleElementArray() {
+        int[] array = {42};
+        int result = findingArraysMedian.median(array);
+        assertEquals(1, result);
+    }
+
+    @Test
+     void testFindMedianPosition_TwoElementArray() {
```

Comment:
Basically the next tests are one and the same code, you could parametrise them if you want, check this - https://www.baeldung.com/parameterized-tests-junit-5